Core/include/Acts/Utilities/JacobianHelpers.hpp

```diff
@@ -0,0 +1,68 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Utilities/VectorHelpers.hpp"
+
+namespace Acts {
+
+/// @brief Calculates the Jacobian for spherical to free
+///        direction vector transformation
+///
+/// @note We use the direction vector as an input because
+///       the trigonometric simplify that way
+///
+/// @param direction The normalised direction vector
+///
+/// @return The Jacobian d(dir_x, dir_y, dir_z) / d(phi, theta)
+///
+inline ActsMatrix<3, 2> sphericalToFreeDirectionJacobian(
```

Comment:
Should this be inline or could it go to a compilation unit?

Reply:
I think if it lives in the header we need the inline to prevent having the same symbol in multiple compilation units

I inlined it to get around possible performance losses